app/javascript/components/recipes/new/create-recipes.vue

```diff
@@ -70,32 +71,117 @@
         @click="create"
       />
     </div>
+
+    <base-modal
+      @ok="addIngredientstoRecipe"
+      @cancel="toggleAddIngredientModal"
+      v-if="showingAddIngredientModal"
+      :title="$t('msg.ingredients.add')"
+      :ok-button-label="$t('msg.add')"
+      :cancel-button-label="$t('msg.cancel')"
+    >
+      <div class="flex f-row w-full">
+        <div class="mr-10">
+          <search-ingredient-list 
+            :ingredients="my_ingredients"
+            @add-ingredient="addIngredient"
+          />
+        </div>
+        <selected-ingredients 
+          :elements="selectedIngredients"
+          :svg="{sixdots: true, cancel: false, menu_recipe: true}"
+          :input="false"
+          :placeholder="'-'"
+          @delete-ingredient="deleteIngredient"
+        /> 
+      </div>
+    </base-modal>
+
   </div>
 </template>
 
 <script>
 import List from './list.vue';
+import BaseModalComponent from '../../base/base-modal.vue';
+import SelectedIngredients from './selected-ingredients.vue';
+import SearchIngredientList from './search-ingredient-list.vue';
+import { getIngredients, postIngredient, deleteIngredient, editIngredient } from './../../../api/ingredients.js';
 
 export default {
   components: {
     List,
+    BaseModalComponent,
+    SelectedIngredients,
+    SearchIngredientList,
   },
   props: {
     objects: { type: Array, required: true },
   },
+  data() {
+    return {
+      showingAddIngredientModal: false,
+      selectedIngredients: [],
+      my_ingredients: [],
+      ingredients: [],
+    }
+  },
+  async created() {
+    try {
+      const response = await getIngredients();
+      this.my_ingredients = response.data.data.map((element) => ({
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>  Identifier 'my_ingredients' is not in camel case                              camelcase